pkgs/native_assets_builder/test/build_runner/helpers.dart

```diff
@@ -19,7 +19,7 @@ Future<void> runPubGet({
 }) async {
   final result = await runProcess(
     executable: Uri.file(Platform.resolvedExecutable),
-    arguments: ['pub', '--no-analytics', 'get'],
+    arguments: ['pub', '--suppress-analytics', 'get'],
```

Comment:
Could you add a comment on why you suppress analytics here?

Reply:
It was added in https://github.com/dart-lang/native/pull/69/commits/a01cd38a0cc27b6895b19c596f5d9e8bce3fc936.

I don't remember exactly why, maybe because it would ask the message to opt in causing test logs to be verbose.